src/fileActions.jsx

```diff
@@ -237,3 +247,70 @@ export const RenameItemModal = ({ path, setPath, selected }) => {
         </Modal>
     );
 };
+
+export const CreateLinkModal = ({ currentPath, files, selected }) => {
+    const Dialogs = useDialogs();
+    const [originalName, setOriginalName] = useState(selected?.name || "");
+    const [newName, setNewName] = useState("");
+    const [type, setType] = useState("symbolic");
+    const [errorMessage, setErrorMessage] = useState(undefined);
+
+    const createLink = () => {
+        const options = { err: "message", superuser: "try" };
+        cockpit.spawn(["ln", ...(type === "symbolic" ? ["-s"] : []), currentPath + originalName.slice(originalName.lastIndexOf("/") + 1), currentPath + newName], options)
+                .then(Dialogs.close, (err) => { setErrorMessage(err.message) });
+    };
+
+    return (
+        <Modal
+          position="top"
+          title={_("New link")}
+          isOpen
+          onClose={Dialogs.close}
+          footer={
+              <>
+                  <Button variant="primary" onClick={createLink}>{_("Create link")}</Button>
+                  <Button variant="link" onClick={Dialogs.close}>{_("Cancel")}</Button>
+              </>
+          }
+        >
+            <Stack>
+                {errorMessage !== undefined &&
+                <InlineNotification
+                  type="danger"
+                  text={errorMessage}
+                  isInline
+                />}
+                <Form isHorizontal>
+                    <FormGroup label={_("Original")}>
+                        <div id="create-link-original-wrapper">
+                            <FileAutoComplete
+                              onChange={setOriginalName} placeholder={_("Path to file")}
+                              superuser="try" value={currentPath + originalName}
+                              id="create-link-original"
+                            />
+                        </div>
+                    </FormGroup>
+                    <FormGroup label={_("New")}>
+                        <TextInput
+                          value={newName} onChange={setNewName}
```

Comment:
In new Patternfly 5 the onChange takes a callback with first parameter the event and second the value.

(_, value) => setNewName(value)


The reason it does not fail locally for you is because you have not rebased on top of your Patternfly PR but the CI does.